src/ZulipMobile.js

```diff
@@ -32,14 +33,16 @@ export default class ZulipMobile extends Component {
 
     return (
       <Provider store={store}>
-        <IntlProvider
-          key={locale}
-          locale={locale}
-          textComponent={Text}
-          messages={messages[locale]}
-        >
-          <NavigationContainer />
-        </IntlProvider>
+        <ActionSheetProvider>
```

Comment:
Do we have to have the ActionSheetProvider here? Maybe inside the Chat.js or even MessageList?